typesafe/src/main/scala/zio/config/typesafe/TreeToHoconSupport.scala

```diff
@@ -17,7 +17,11 @@ private[typesafe] trait PropertyTreeFunctions {
     tree match {
       case leaf @ Leaf(_)     => loopLeaf(key, leaf)
       case record @ Record(_) => loopRecord(record)
-      case PropertyTree.Empty => ConfigFactory.empty().root()
+      case PropertyTree.Empty =>
+        key.fold(ConfigFactory.empty().root()) { k =>
+          ConfigFactory.empty().withValue(k, ConfigValueFactory.fromAnyRef(null: Option[String])).root()
+        }
```

Comment:
Ofcourse, if `empty` it is not `x : {}` which corresponds to ` ConfigFactory.empty().root()`.
This was breaking before since we now pattern match on the type of tree we looking at, and it always gave `Record`